README.md

```diff
@@ -1,70 +1,142 @@
 
-## Learn More
+### Install
+Install this project with:
+  `npm install `
+ <p>No special installation needed</p>
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+### Usage
+To run the project, execute the following command:
+`npm start`
 
-### Code Splitting
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+### Run tests
+To run tests, run the following command:
+<p> No test available </p>
+<p>But you can inspect the project's webpages by the browser's developer mode </p>
 
-### Analyzing the Bundle Size
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+### Deployment
+You can deploy this project using:
+<p>Any Browser</p>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Making a Progressive Web App
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
 
-### Advanced Configuration
+## 👥 Authors <a name="authors"></a>
+ **Ibrahim ibu**
+- GitHub: [@githubhandle](https://github.com/ibugithub)
+- Twitter: [@twitterhandle](https://twitter.com/mdibrahimibuu)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ibuu)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+- [ ] **[Beautiful Ui Structure]**
+- [ ] **[Fully functional calculator]**
+- [ ] **[Interactive website]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](../../issues/).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+If you like this project please try to contribute or rate it or put a comment
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+I would like to thank to my coding partners.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
 
-### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] the link to the MIT license is not working as expected, please make sure it is working similarly to the template provided by Microverse.